docs/source/tutorial.rst

```diff
@@ -20,206 +20,438 @@ is from UCI's `machine learning repository <http://archive.ics.uci.edu/ml/datase
 
 What You'll Need
 ----------------
-To run this tutorial, you'll need to:
 
- - Install MLflow (via ``pip install mlflow``)
- - Install `conda <https://conda.io/docs/user-guide/install/index.html#>`_
- - Clone (download) the MLflow repository via ``git clone https://github.com/mlflow/mlflow``
- - `cd` into the ``examples`` directory within your clone of MLflow - we'll use this working
-   directory for running the tutorial. We avoid running directly from our clone of MLflow as doing
-   so would cause the tutorial to use MLflow from source, rather than your PyPi installation of
-   MLflow.
+.. plain-section::
 
+    .. container:: python
+
+      To run this tutorial, you'll need to:
+
+       - Install MLflow (via ``pip install mlflow``)
+       - Install `conda <https://conda.io/docs/user-guide/install/index.html#>`_
+       - Clone (download) the MLflow repository via ``git clone https://github.com/mlflow/mlflow``
+       - `cd` into the ``example`` directory within your clone of MLflow - we'll use this working
+         directory for running the tutorial. We avoid running directly from our clone of MLflow as doing
+         so would cause the tutorial to use MLflow from source, rather than your PyPi installation of
+         MLflow.
+
+    .. container:: R
```

Comment:
Can we sync the language selection for all blocks? I.e. when user clicks R, the rest of the tutorial switches to R.

Reply:
Yes, I had a mental not for this. It's annoying to have to switch per section. I can also add a `#language=r` or `#language=python` to the URL to be able to share sections in the correct language.